### ISSUE TITLE
TiKV write send event failed logs frequently

### ISSUE THREAD

**Issue by 3pointer** [pingcap/ticdc#610]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.
when cdc is broken, TiKV write ERROR log `send event failed` too frequently.

2. What did you expect to see?
maybe a few errors of "send event failed"
or 
handle this error correctly.

3. What did you see instead?
```
[2020/06/02 14:34:43.632 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:54894]
[2020/06/02 14:34:43.632 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:54894]
[2020/06/02 14:34:43.633 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:54894]
[2020/06/02 14:34:43.633 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:53258]
[2020/06/02 14:34:43.634 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:54836]
[2020/06/02 14:34:43.635 +08:00] [ERROR] [delegate.rs:135] ["send event failed"] [downstream=ipv4:xxx:54894]
```

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        4.0.0-rc.2
    - TiCDC version (execute `cdc version`):

        
```
Release Version: v4.0.0-rc.1-25-g961d4c8
Git Commit Hash: 961d4c87562ccab8273c6aa220e665664862e9f6
Git Branch: HEAD
UTC Build Time: 2020-05-11 19:52:39
Go Version: go version go1.13 linux/amd64
```

**Comment by 5kbpers** [comment]
How long did this log flush continually?

**Comment by overvenus** [comment]
https://github.com/tikv/tikv/pull/8623 may fix this issue.

**Comment by amyangfei** [comment]
Please help to confirm whether this issue has been fixed @5kbpers

**Comment by overvenus** [comment]
Fixed by tikv/tikv#8623.
